pkg/core/resources/manager/manager.go

```diff
@@ -62,6 +66,26 @@ func (r *resourcesManager) Delete(ctx context.Context, resource model.Resource,
 	return r.Store.Delete(ctx, resource, fs...)
 }
 
+func (r *resourcesManager) DeleteAll(ctx context.Context, fs ...store.DeleteAllOptionsFunc) error {
+	opts := store.NewDeleteAllOptions(fs...)
+
+	for _, typ := range r.Registry.ListTypes() {
```

Comment:
I don't agree that logic that goes through the list of all supported types and calls `DeleteAll` on each of them should be a part of generic `ResourcesManager`.

By design, `ResourceStore` was a store for a single resource type. `ResourceManager` was introduced to complement `ResourceStore` with type-specific business logic.

In a generic case, business logic of `DeleteAll` is to delete all entities of a single resource type. It doesn't make sense to delete objects of all types when `DeleteAll` is called for `TrafficLog` or `TrafficPermission`.

On the other hand, delete objects of all types is business logic specific to `Mesh`. So, I would suggest to move that code into `MeshManager`